task/06-conditions-n-loops-tasks.js

```diff
@@ -150,7 +183,10 @@ function doRectanglesOverlap(rect1, rect2) {
  *   
  */
 function isInsideCircle(circle, point) {
-    throw new Error('Not implemented');
+  const diff = Math.pow((circle.center.x - point.x), 2) + Math.pow((circle.center.y - point.y), 2);
+    if (diff < Math.pow(circle.radius, 2)) {
+        return true;
+    } else return false;
```

Comment:
https://javascript.info/logical-operators
We can just ` return diff < Math.pow(circle.radius, 2)`